### DISCUSSION TITLE
Accidental delete of Supabase records

### DISCUSSION THREAD

**Discussion by Ecksdee03** [supabase/supabase#26983] [Questions]
Hi, is there a way for me to restore deleted records on Supabase? I accidentally deleted the entire table along with its records, but as I am on free plan there are no backup or PTO options available. I have contacted support already but was wondering if there is anything I can do to restore the records

**Comment by GaryAustin1** [comment]
You can pay for pro and you will have access to your backups.  You are committed for at least a month.
You should start your own back up with a 3rd party tool if you are doing more than basic testing on a free plan.

**Reply by Ecksdee03** [reply]
If I upgrade to pro now, will I still get access to the previous backups before my subscription?

**Reply by GaryAustin1** [reply]
Others have said 7 days worth there.

**Reply by edvinPL** [reply]
> Others have said 7 days worth there.

This is crazy good
and IT WORKS
My DB got wiped by bad SQL and after payment - backups were there!
Phew!
